packages/flutter_tools/lib/src/globals.dart

```diff
@@ -14,6 +14,9 @@ Cache get cache => Cache.instance;
 Config get config => Config.instance;
 Artifacts get artifacts => Artifacts.instance;
 
+/// Disables the trace logger entirely.
+bool disableTrace = false;
```

Comment:
what's the purpose of this?

Reply:
saves 5-10 ms

Reply:
This is breaking some tests so I will deal with it separately